.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+sudo: true
```

Comment:
do we need sudo?

Reply:
I get this error in travis, if I don't use sudo:

[ERROR] Plugin org.apache.maven.plugins:maven-resources-plugin:2.6 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.apache.maven.plugins:maven-resources-plugin:jar:2.6: Could not transfer artifact org.apache.maven.plugins:maven-resources-plugin:pom:2.6 from/to central (https://repo.maven.apache.org/maven2): Access denied to: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-resources-plugin/2.6/maven-resources-plugin-2.6.pom , ReasonPhrase:Forbidden. -> [Help 1]

Reply:
Did you read what that message means?

Reply:
It says access was denied. I used sudo as a workaround suggested for the same problem [here](https://github.com/tautschnig/cbmc/commit/ef608d2a167a64971d0e68748be14f2a61f26f52)

Reply:
I opened an issue [here](https://github.com/travis-ci/travis-ci/issues/10090#issue-357721251) for this bug.

Reply:
oh that's going to probably be a duplicate on their side.  I meant documenting it on this repository's issue tracker

Reply:
ok it's [here](https://github.com/in-toto/in-toto-java/issues/3#issue-357747580) now